omnibus/config/software/datadog-agent-integrations-py2.rb

```diff
@@ -240,6 +240,12 @@
         end
       end
 
+      # Copy SNMP profiles
+      profiles = "#{check_dir}/datadog_checks/#{check}/data/profiles"
+      if File.exist? profiles
+        copy profile, "#{check_conf_dir}/"
```

Comment:
the `profile` var isn't defined (see errors in gitlab)

Reply:
Fixed, thanks.